bgpd/bgp_route.c

```diff
@@ -11847,6 +11872,10 @@ DEFPY (show_ip_bgp_json,
        "RT translated VPNv6 route filtering (well-known community)\n"
        "RT translated VPNv4 route filtering (well-known community)\n"
        "Exact match of the communities\n"
+       "RPKI route types\n"
```

Comment:
Can these descriptions be the same as for show_ip_bgp_route?

Reply:
fixed